core/src/main/java/com/rim/IceField/Iceberg.java

```diff
@@ -0,0 +1,127 @@
+package com.rim.IceField;
+
+
+import java.util.ArrayList;
+
+public class Iceberg {
+    private boolean isStable;
+    private int num;
+    //private int numOfPlayers; NO NEED, I JUST ADD THE LIST OF PLAYERS ON THE ICEBERG
+    private String type;
+    private int maxNumOfPlayers;
+    private boolean hasIgloo;
+    private ArrayList<PlayerBase> drowningPlayers = new ArrayList<PlayerBase>();
+    private ArrayList<PlayerBase> currentPlayers = new ArrayList<PlayerBase>(); //List I added
+    private ArrayList<Iceberg> neighborIcebergs = new ArrayList<Iceberg>();
+    private int amountOfSnow;
+    private ItemBase item;
+
+public Iceberg(boolean isStable,int num,String type,int maxNumOfPlayers,boolean hasIgloo, int amountOfSnow, ItemBase item)
+{
+    //FINISH CONSTRUCTOR
+}
+    public void setItem(ItemBase item)
+    {
+        this.item = item;
+    }
+    public ItemBase getItem()
+    {
+        return this.item;
+    }
+    public void setAmountOfSnow(int n)
+    {
+        this.amountOfSnow = n;
+    }
+
+    public int getAmountOfSnow()
+    {
+        return this.amountOfSnow;
+    }
+
+    public ArrayList<Iceberg> getNeighborIcebergs()
+    {
+        return this.neighborIcebergs;
+    }
+
+    public ArrayList<PlayerBase> getCurrentPlayers()
+    {
+        return this.currentPlayers;
+    }
+
+
+    public ArrayList<PlayerBase> getDrowningPlayers()
+    {
+        return this.drowningPlayers;
+    }
+
+    public void setHasIgloo(boolean b)
+    {
+        this.hasIgloo = b;
+    }
+    public boolean getHasIgloo()
+    {
+        return this.hasIgloo;
+    }
+
+    public void setMaxNumOfPlayers(int n)
```

Comment:
Does the maxNumOfPlayers change? If you initialize it in the constructor, seems like you don't need this function